quantum/quantum.h

```diff
@@ -150,7 +150,8 @@ extern layer_state_t default_layer_state;
     typedef uint8_t pin_t;
 
     #define PIN_ADDRESS(p, offset)  (_SFR_IO8(ADDRESS_BASE + ((p) >> PORT_SHIFTER) + (offset)))
-    #define setPinInput(pin)        (PIN_ADDRESS(pin, 1) &= ~_BV((pin) & 0xF))
+    #define setPinInput(pin)        (PIN_ADDRESS(pin, 1) &= ~_BV((pin) & 0xF), \
+                                     PIN_ADDRESS(pin, 2) =  _BV((pin) & 0xF))
```

Comment:
I think this should be `&=`.
```suggestion
                                     PIN_ADDRESS(pin, 2) &=  _BV((pin) & 0xF))
```

Reply:
Indeed I made the change a bit too quickly. It was also missing the bitwise negate ~

Thanks :)